### ISSUE TITLE
Order robotoff questions by arbitrary criteria

### ISSUE THREAD

**Issue by teolemon** [openfoodfacts/robotoff#907]
### What
- Currently It seems we return questions in random order 
- Allow to Order Robotoff questions by arbitrary criteria
  - Timestamp of creation
  - Timestamp of image upload
  - Negative filtering of unwanted labels (en:green-dot)
  - Positive filtering of desirable labels (create a list of Eco-Score related labels)
  - Priority (with a general arbitrary sequence to be defined , eg #1 category (useful for NS and ES), #2 nutrition (useful for NS), #3 brand (useful for SEO), 4 weight (useful for SEO), #5 labels (ranked by usefulness, especially Eco-Score related ones)
  - Priority by objective (to get the Eco-Score, to get the Nutri-Score, to get the vegan status), potentially concatenating robotoff insight, Product Opener field status, to get a JSON to-do list of things to answer/complete.

**Comment by alexgarel** [comment]
@alexfauquette I now this might be a red herring, as hung-games as not so much users, but if we order questions from the most recent and two users use hunger-games at the same time, isn't any risk of conflicts (that they will try to answer same questions at the same time) ?

**Comment by alexfauquette** [comment]
@alexgarel It's already the case with sort by popularity. Since we do not show errors, the annotation will simply be ignored by the server.

If I remember correctly, it returns a message such as it has already been annotated

You can try it yourself by opening the following page in two distinct tabs
https://hunger.openfoodfacts.org/questions?type=label&sorted=true

First annotation:
```json
{"status":"updated","description":"the annotation was saved and sent to OFF"}
```
Second annotation:
```json
{"status":"error_already_annotated","description":"the insight has already been annotated"}
```

**Comment by alexgarel** [comment]
Ok, but it's a bit frustrating to know that we might loose contributors work ?
Shan't hunger-game introduce some randomness (eg start at page random[1-20]) to avoid this ?

**Comment by alexfauquette** [comment]
Yes, that's feasible. Maybe we can wait to have more users before doing this kind of improvement
